grpc/channelz/v1/channelz.proto

```diff
@@ -435,9 +436,14 @@ service Channelz {
 }
 
 message GetTopChannelsRequest {
-  // start_channel_id indicates that only channels at or above this id should be
+  // start_channel_id indicates that only channels above this id should be
   // included in the results.
   int64 start_channel_id = 1;
+
+  // If non-zero, the server will return a page of results whose
+  // cardinality is equal to or less than this value. If zero, the
+  // server will choose a reasonable page size.
+  uint32 limit = 2;
 }
 
 message GetTopChannelsResponse {
```

Comment:
not your change, but I think this doc needs to say that there must be at least one channel returned, or else end must be set.  (to avoid a possible infinite loop where the client requests channels, but the server returns none and doesn't set end.